### PR TITLE
Adding french translations to galileo.txt

### PR DESCRIPTION
**Pull request by Nemesis31**
Partial translation in french.
Need review and help from frenchies people :)

### CODE REVIEW

**Comment by evandrocoan** [comment]
Thanks. I posted it on the translation thread on the AlliedModders forum https://forums.alliedmods.net/showthread.php?p=2504662#post2504662

File: plugins/addons/amxmodx/data/lang/galileo.txt [hunk 1]
```diff
@@ -1357,3 +1357,139 @@ DMAP_NEXTMAP_VOTE_REMAINING2 = Szavazás indul a!g következő pályáról!y %d!
...
+GAL_ROCK_WAIT_ADMIN = Tu ne peux pas '!grock the vote!y' quand un admin!g  est en ligne!y.
+GAL_ROCK_NEEDMORE = !g%i!t plus de joueurs doivent '!yrockthevote!t' pour commencer un vote.
+GAL_ROCK_SUCCESS = Tu as!t rock the vote!y.
+GAL_RUNOFF_REQUIRED = !tRunoff voting est nécessaire!y car le premier choix!g n'a pas reçu plus de %d%%!t des votes exprimés.
+GAL_RUNOFF_REQUIRED_TOP = Le premier choix n'a pas reçu suffisamment de voix
+GAL_STANDARD_NOTFOUND = Erreur lors de l'ouverture de '%s' pour charger des cartes standards.
+GAL_STANDARD_TOOMANY = Il existe %i cartes standards définies pour le jeu '%s'.
+GAL_STANDARD_UNKNOWNMOD = Aucune carte standard définie pour le jeu '%s'.
+GAL_TIMELEFT = Secondes restantes
+GAL_VOTE_COUNTDOWN = Le vote suivant de la carte commencera dans %i secondes
```
Comment (GoRiLliAz) [inline plugins/addons/amxmodx/data/lang/galileo.txt:1469]: +GAL_VOTE_COUNTDOWN = Le vote suivant de la carte commencera dans %i

File: plugins/addons/amxmodx/data/lang/galileo.txt [hunk 2]
```diff
@@ -1357,3 +1357,139 @@ DMAP_NEXTMAP_VOTE_REMAINING2 = Szavazás indul a!g következő pályáról!y %d!
...
+GAL_ROCK_ENOUGH = Assez de joueurs ont!t rock the vote!y qu'un vote pour la prochaine carte commencera.
+GAL_ROCK_FAIL_ALREADY = Tu as!t déjà rock the vote!y.
+GAL_ROCK_FAIL_INPROGRESS = Un vote est!t déjà en cours!y.
+GAL_ROCK_FAIL_PENDINGVOTE = Il y a un vote!t déjà programmé dans moins de!g 2 minutes!y.
+GAL_ROCK_FAIL_TOOSOON = Tu dois attendre approximativement!t %i minutes!y avant de pouvoir '!grock the vote!y'.
+GAL_ROCK_FAIL_TOOSOON_ROUNDS = Tu dois attendre approximativement!t %i rounds!y avant de pouvoir '!grock the vote!y'.
+GAL_ROCK_FAIL_TOOSOON_FRAGS = Tu dois attendre approximativement!t %i frags!y avant de pouvoir '!grock the vote!y'.
+GAL_ROCK_FAIL_VOTEOVER = Le résultat!t du vote!g a déjà été déterminé.
+GAL_ROCK_WAIT_ADMIN = Tu ne peux pas '!grock the vote!y' quand un admin!g  est en ligne!y.
+GAL_ROCK_NEEDMORE = !g%i!t plus de joueurs doivent '!yrockthevote!t' pour commencer un vote.
```
Comment (GoRiLliAz) [inline plugins/addons/amxmodx/data/lang/galileo.txt:1461]: +GAL_ROCK_NEEDMORE = !g%i!t  joueurs de plus doivent '!yrockthevote!t' pour commencer un vote.

**Review by GoRiLliAz — COMMENTED** [review]
Its better

**Pull request merged**